modules/pam_timestamp/pam_timestamp.c

```diff
@@ -449,14 +456,22 @@ pam_sm_authenticate(pam_handle_t *pamh, int flags, int argc, const char **argv)
 			return PAM_AUTH_ERR;
 		}
 
+#ifdef WITH_OPENSSL
+		if (hmac_size(pamh, debug, &maclen)) {
+			pam_syslog(pamh, LOG_CRIT, "Unable to get hmac size");
```

Comment:
I would drop this log as there will be already a log message from the hmac_size() call. Too much logging is not good.